ui/app/src/components/Dialogs/ConfirmDialog.tsx

```diff
@@ -75,32 +121,43 @@ export default function ConfirmDialog(props: ConfirmDialogProps) {
       disableBackdropClick={props.disableBackdropClick}
       disableEnforceFocus={props.disableEnforceFocus}
       hideBackdrop={props.hideBackdrop}
+      className={classes.dialog}
     >
       <ConfirmDialogWrapper {...props} />
     </Dialog>
   );
 }
 
 function ConfirmDialogWrapper(props: ConfirmDialogProps) {
-  const { onOk, onCancel, onDismiss, body, title, children } = props;
+  const { onOk, onCancel, body, title, children } = props;
   const { formatMessage } = useIntl();
+  const classes = confirmDialogStyles();
```

Comment:
Pass down the classes from the `ConfirmDialog` to avoid appending these set of styles twice.